03-Custom-Actions-In-Python/06-Improving-Performance.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>Maybe say "time complexity" instead of just "complexity".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>